### PR TITLE
Holy/Comet combo

### PR DESCRIPTION
**Pull request by kaedys**
- Implemented PictomancerHolyCometCombo using the Monochrome Tones buff rather than the (not yet implemented) gauge.
- Corrected description of the two Pictomancer Subtractive Palette combos to reflect their actual behavior.

I'm not even sure the gauge implementation will have a Black Paint counter.  Mechanically, the Black Paint occurs just when you have White Paint _and_ also have the Monochrome Tones buff.  This implementation also has the effect of changing Holy to Comet if you have the buff but do not yet have any paint, as the next paint generated will still be counted as black paint and Comet will still be required over Holy.  This avoids the button suddenly swapping on the bar once the paint is generated.

Basically, we don't even need the gauge for this one, because the Monochrome Tones buff is the one that actually mechanically controls whether Comet or Holy is available.  You can't cast Comet without that buff, and you can't cast Holy with it.

Also corrected the descriptions of the Subtractive Palette combos, because how they're implemented, they replace Blizzard with Fire when Subtractive is inactive, rather than replacing Fire with Blizzard when Subtractive is active, as they previously stated.

### CODE REVIEW

File: XIVComboExpanded/CustomComboPreset.cs
Comment (MKhayle) [inline XIVComboExpanded/CustomComboPreset.cs:1]: oof, that one is on me, yes. thanks!
